library-backend/library-api/src/main/java/org/internship/library/api/BookRepository.java

```diff
@@ -0,0 +1,33 @@
+package org.internship.library.api;
+
+/**
+ * This interface defines the behaviour of Book repository
+ */
+public interface BookRepository {
+
+    /**
+     * Method without implementation to find a book
```

Comment:
We should not write in the javadoc the obvious thing that the method is empty.

We should point out what this method can do for a Repository.

e.g. Finds the book by the given id.

Besides that you can add what's returned (or if an exception is thrown) when there's no book with that id.

Also, I think we've discussed that for methods that return a Book from the DB, or must find something, we use Optional<Book>.